app/views/policies/_personal_detail_show.html.slim

```diff
@@ -0,0 +1,14 @@
+table.table
+  tbody
+  tr
```

Comment:
@GauravMakwana-BTC : there should be a tab between tbody & tr, update it below as well